Tests/MMELocationManagerTests.m

```diff
@@ -18,14 +18,46 @@ @interface MMELocationManager (Spec) <CLLocationManagerDelegate>
 
 @end
 
+// MARK: - Mock
+
+@interface MockCLLocationManager: CLLocationManager
```

Comment:
TODO: Remove. This was an attempt at mocking out location manager method calls which ran into the issue of not working due to a cached location in the location manager leaving distance filtering flaky.